tests/assertions/enums.n

```diff
@@ -10,7 +10,7 @@
 	assert type gameState : state
 }
 
-{
+let _ = [] -> () {
```

Comment:
I don't think this runs the function though

Reply:
Oop, fixed it